src/main/java/se/sundsvall/teamssender/api/TeamsSenderResource.java

```diff
@@ -26,7 +27,9 @@ public TeamsSenderResource(TeamsSenderService teamsSenderService) {
 	@PostMapping("/teams/messages")
 	@Operation(summary = "Send a message in Microsoft Teams",
 		responses = {
-			@ApiResponse(responseCode = "200", description = "Message sent successfully", useReturnTypeSchema = true),
+			@ApiResponse(responseCode = "200",
+				description = "Message sent successfully",
+				content = @Content(schema = @Schema(implementation = SendTeamsMessageResponse.class))),
```

Comment:
`useReturnTypeSchema = true` is to be preferred over the "content"-attrbute i.m.h.o

(i.e. just leave it `@ApiResponse`-annotation as it was)